lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -501,8 +501,11 @@ defmodule ExDoc.Formatter.HTML.Autolink do
     end
   end
 
+  defp default_lib_dirs(),
+    do: default_lib_dirs(:elixir) ++ default_lib_dirs(:erlang)
+
   defp default_lib_dirs(:elixir),
-    do: elixir_lib_dirs()
+    do: elixir_lib_dirs() ++ hex_lib_dirs()
```

Comment:
It is fine to keep this hardcoded for now, since Hex is an important part of the community, but we may want to make this configurable in the future.